### PR TITLE
feat: add Cloudflare Workers module and hono adapters

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Props to the person without name on Telegram, see https://t.me/grammyjs/104653.

The holo adapter is duplicated because the `.web.ts` files currently are not included in the Deno and Node builds. Should we add these dependencies in order to be DRY here?

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
We added the dependency between the files. They were included in the releases anyway. The holo adapter now isn't duplicated anymore.

File: src/convenience/frameworks.web.ts [hunk 1]
```diff
@@ -34,6 +34,61 @@ const cloudflare = (
     };
 };
 
+/** Native CloudFlare workers (module worker) */
+const cloudflare_module = (
```
Comment (KnorpelSenf) [inline src/convenience/frameworks.web.ts:38]:
```suggestion
const cloudflareModule = (
```

File: src/convenience/frameworks.web.ts [hunk 2]
```diff
@@ -43,4 +98,9 @@ const worktop = (req: any, res: any) => ({
     unauthorized: () => res.send(401, "secret token is wrong"),
 });
 
-export const adapters = { cloudflare, worktop };
+export const adapters = {
+    cloudflare,
+    "cloudflare-mod": cloudflare_module,
```
Comment (KnorpelSenf) [inline src/convenience/frameworks.web.ts:103]:
```suggestion
    "cloudflare-mod": cloudflareModule,
```

**Review by dcdunkan — APPROVED** [review]
LGTM

**Comment by KnorpelSenf** [comment]
Thanks for the review! ❤️

**Pull request merged**